server/Makefile

```
@@ -1,8 +1,10 @@
 VERSION ?= "dev"
 TRACETEST_ENV ?= "dev"
+POKE_API?= "dev"
```

a good default for this would be `demo-api`:
```suggestion
POKE_API?= "demo-api:8082"
```

Also, it would be awesome to inject the same but for the grpc container